publish-report/action.yml

```diff
@@ -43,7 +43,7 @@ runs:
 
     - name: Write report link to summary
       shell: bash
-      run: echo '<a href="https://storage.googleapis.com/releng-pipeline-artifacts-dev/${EVENT_REPOSITORY_NAME}/${EVENT_NUMBER}/Grafana-v${GRAFANA_VERSION}/index.html">Browse the Playwright report for Grafana-v${GRAFANA_VERSION}</a>' >> $GITHUB_STEP_SUMMARY
+      run: echo '<a href="https://storage.googleapis.com/releng-pipeline-artifacts-dev/$EVENT_REPOSITORY_NAME/$EVENT_NUMBER/Grafana-v$GRAFANA_VERSION/index.html">Browse the Playwright report for Grafana-v$GRAFANA_VERSION</a>' >> $GITHUB_STEP_SUMMARY
```

Comment:
this is going to store the variable names in GITHUB_STEP_SUMMARY, is that what we want? I would thought that we may want to do (use `"` so the variables are replaced):

```
      run: echo "<a href=\"https://storage.googleapis.com/releng-pipeline-artifacts-dev/${EVENT_REPOSITORY_NAME}/${EVENT_NUMBER}/Grafana-v${GRAFANA_VERSION}/index.html\">Browse the Playwright report for Grafana-v${GRAFANA_VERSION}</a>" >> $GITHUB_STEP_SUMMARY
```

Reply:
ah, it's good now :) not sure why github showed me the wrong commit